clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10394,6 +10393,9 @@ static void getOtherSPIRVTransOpts(Compilation &C,
               ",+SPV_INTEL_tensor_float32_conversion"
               ",+SPV_INTEL_optnone"
               ",+SPV_KHR_non_semantic_info";
+  if (C.getDriver().IsFPGAHWMode() || C.getDriver().IsFPGAEmulationMode())
```

Comment:
@mdtoguchi was IsFPGAEmulationMode function fixed? I remember it was doing funny things, basically it returns true by default even for JIT.

Reply:
Nvm, I see https://github.com/intel/llvm/issues/4061 is marked as resolved.